lib/iron_bank/local.rb

```diff
@@ -81,5 +83,14 @@ def file_path
         IronBank.configuration.export_directory
       )
     end
+
+    # :reek:UtilityFunction
```

Comment:
👍 I think bypassing reek warning here make sense, it makes the `#where` method more readable.